### DISCUSSION TITLE
How to Show and Hide Content Scrolling Up and Down in FlashList?

### DISCUSSION THREAD

**Discussion by ahmtxonly** [Shopify/flash-list#688] [Q&A]
Hello,

How we can do this?

![1_4MPbHIzUmKO57gKaizWYeA](https://user-images.githubusercontent.com/36918073/204215878-4a58225f-a307-4a50-85be-8e1c041be7b4.gif)

**Comment by Bayramito** [comment]
it is called collapsing animation...

there are tons of example on the internet for that, simply pick one of them (prefer with reanimated 2/3) then convert your flashlist to AnimatedFlashList and use it using onScroll method
